src/oscar/apps/catalogue/abstract_models.py

```diff
@@ -128,6 +128,8 @@ class AbstractCategory(MP_Node):
 
     name = models.CharField(_('Name'), max_length=255, db_index=True)
     description = models.TextField(_('Description'), blank=True)
+    meta_title = models.CharField(_('Meta title'), max_length=255, blank=True)
+    meta_description = models.TextField(_('Meta description'), blank=True)
```

Comment:
Please also allow null=True that will avoid having to make a copy of the entire database during the migration.